core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -451,7 +451,7 @@ protected Scope visitInsert(Insert insert, Optional<Scope> scope)
 
```

Comment:
Can you please first add a commit that reproduces the problem and then with second commit you could fix the problem so we could see how it affects the tests from the first commit?

Reply:
Added tests for it.